_sass/blocks/explore/_filters.scss

```diff
@@ -1,6 +1,16 @@
 
+.filters-wrapper_outer {
+  position: relative;
+}
+
 .filters-wrapper {
+  background-color: transparent;
+  max-width: 654.609px;
```

Comment:
Properties should be ordered background-color, margin-left, max-width, padding, position, width, z-index